tests/playwright/django-ory-cloud.spec.ts

```diff
@@ -0,0 +1,77 @@
+import { test, expect, Page } from "@playwright/test"
+import { randomEmail, randomString } from "./helpers"
+
+const login = async (page: Page) => {
+  const email = randomEmail()
+  await page.fill('[name="traits.email"]', email)
+  await page.fill('[name="password"]', randomString())
+  await page.click('[value="password"]')
+  return email
+}
+
+test.describe("django-ory-cloud", () => {
+  for (const app of [
+    { url: "http://localhost:4000/", name: "django" },
+  ]) {
+    test.describe(app.name, async () => {
```

Comment:
As an alternative to the suggestion above, you could just hard-code the app name here:

```suggestion
    test.describe("django", async () => {
```